crates/uv/src/commands/project/version.rs

```diff
@@ -168,8 +170,41 @@ pub(crate) async fn project_version(
                 }
             },
         }
-    } else if let Some(bump) = bump {
-        Some(bumped_version(&old_version, bump, printer)?)
+    } else if !bump.is_empty() {
+        // Sort the given commands so the user doesn't have to care about
+        // the ordering of `--bump minor --bump major` (only one ordering is ever useful)
+        bump.sort();
+
+        // Apply all the bumps
+        let mut new_version = old_version.clone();
+        for bump in &bump {
+            let command = match *bump {
+                VersionBump::Major => BumpCommand::BumpRelease { index: 0 },
+                VersionBump::Minor => BumpCommand::BumpRelease { index: 1 },
+                VersionBump::Patch => BumpCommand::BumpRelease { index: 2 },
+                VersionBump::Alpha => BumpCommand::BumpPrerelease {
+                    kind: PrereleaseKind::Alpha,
+                },
+                VersionBump::Beta => BumpCommand::BumpPrerelease {
+                    kind: PrereleaseKind::Beta,
+                },
+                VersionBump::Rc => BumpCommand::BumpPrerelease {
+                    kind: PrereleaseKind::Rc,
+                },
+                VersionBump::Post => BumpCommand::BumpPost,
+                VersionBump::Dev => BumpCommand::BumpDev,
+                VersionBump::Stable => BumpCommand::MakeStable,
+            };
+            new_version.bump(command);
+        }
+
+        if !allow_decreases && new_version < old_version {
```

Comment:
Perhaps a bit pedantic, but I wonder if this would be better as `allow_non_increasing` with `new_version <= old_version`. That is, if my `--bump` _doesn't_ increase the version number in some way, then I think I'd want to be bleated at. So if the version stays equal, for example, that would error here.

(`allow_non_increasing` is kind of a mouthful, so it might make sense to keep the `allow_decreases` name and just have it be slightly imprecise.)